src/borg/archiver.py

```diff
@@ -3106,6 +3106,18 @@ def define_borg_mount(parser):
         exclusive because the data is not actually compressed and deduplicated during a dry run.
 
         See the output of the "borg help patterns" command for more help on exclude patterns.
+        To give an example for --exclude-from:
+        ```shell
+        $ cat exclude-file.txt
+        # Comment line
+        /home/*/junk
+        *.tmp
+        One file with spaces
+        ```
+        Then the command borg ``create --exclude-from exclude-file.txt`` will
+        delete all files excluding /home/*/junk, *.tmp and the file named
+        ``One file with spaces``
+
```

Comment:
be careful when using markup in archiver.py:

while the rst / sphinx docs (-> online, html, pdf) support rather powerful markup, the stuff that `borg help` outputs should also look ok (and guess we only support a rather limited subset of the markup for that).

for each command, there is also a separate .rst.inc (.inc.rst?) file with examples, that could also be used.

one could also check that "borg help patterns" text, considering it is already mentioned a line above and that we need to refer to such an example for multiple borg commands, at least create, recreate and mount - so putting it just into the create help is maybe not the best solution and also causes duplication.

Reply:
I'll put it in the borg help patterns then :)

Reply:
is this still needed?